src/adapters/supabase/helpers/tables/locations.ts

```diff
@@ -0,0 +1,53 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Super } from "./super";
+import { Context } from "../../../../types/context";
+
+// currently trying to save all of the location metadata of the event.
+// seems that focusing on the IssueComments will provide the most value
```

Comment:
Please let's just save the URL only. I want to get rid of the locations table because it is a headache to deal with and provides little value.